docs/kubernetes/kctlr-manage-bigip-objects.rst

```diff
@@ -1,317 +1,204 @@
-
-``<namespace>_<configmap-name>``.
-
-   For example, ``default_http.pool_only``.
+You can use the |kctlr| to create and manage BIG-IP pools that aren't attached to a front-end virtual server (:dfn:`unattached pools`).
 
 .. important::
 
-   Before creating unattached pools, make sure the BIG-IP system has another way to route traffic to the Service(s), such as an `iRule`_ or a `local traffic policy`_. After creating an unattached pool for a Service, use the BIG-IP config utility to add the pool members to the iRule or traffic policy. This ensures proper handling of client connections to your back-end applications.
+   Before you create unattached pools, make sure the BIG-IP system has another way to route traffic to the Service(s), such as `iRules <https://devcentral.f5.com/irules>`_ or `local traffic policies`_.
+
+   You can also :ref:`use an IPAM system <kctlr-ipam>` to allocate IP addresses and create virtual servers for unattached pools.
 
 .. _kctlr-create-unattached-pool:
 
 Create an unattached pool
 `````````````````````````
 
-#. Create an :ref:`F5 resource <k8s-f5-resources>` `ConfigMap`_, but leave out the ``bindAddr`` field.
+#. Create an :ref:`F5 resource <k8s-f5-resources>` **without the** :code:`bindAddr` **property**.
 
    .. literalinclude:: /kubernetes/config_examples/f5-resource-pool-only-example.configmap.yaml
       :linenos:
       :emphasize-lines: 25-27
 
    :fonticon:`fa fa-download` :download:`f5-resource-pool-only-example.configmap.yaml </kubernetes/config_examples/f5-resource-pool-only-example.configmap.yaml>`
 
-#. Upload the ConfigMap to the Kubernetes/OpenShift API server.
-
-   .. code-block:: console
-      :caption: kubectl
-
-      kubectl create -f f5-resource-pool-only-example.configmap.yaml [--namespace=<service-namespace>]
-      configmap "http.pool_only" created
+#. :ref:`Update the API server <kctlr upload resource api server>`.
 
-   .. code-block:: console
-      :caption: openshift cli
+#. Use the BIG-IP configuration utility or TMSH to add the pool members to an iRule or traffic policy with the correct routing rules for your application **--OR--**
 
-      oc create -f f5-resource-pool-only-example.configmap.yaml [--namespace=<service-namespace>]
-      configmap "http.pool_only" created
-
-.. important::
+   :ref:`kctlr-attach-pool-vs` **--OR--**
 
-   Don't forget to attach the pool to the iRule or traffic policy on the BIG-IP system that knows how to route traffic for the Service.
+   :ref:`kctlr-ipam`.
 
-.. _kctlr-attach-pool-vs:
+.. _kctlr-ipam:
 
-Attach a pool to an existing virtual server
-```````````````````````````````````````````
+Attach pools to virtual servers using IPAM
+``````````````````````````````````````````
 
-#. Add the desired :code:`bindAddr` to the F5 virtual server resource definition.
+.. include:: /_static/reuse/k8s-version-added-1_1.rst
 
-   .. tip::
+One benefit of unattached pools is the compatibility with IP address allocation using IPAM.
```

Comment:
I think this and the lower section starting with "To provide even greater benefit" are redundant. The only useful way to use this annotation is for the IPAM system to write the annotation back to the ConfigMap. Otherwise, if I'm a user and reserve an address manually I'll use the bindAddr section manually.